tasks/frontEnd/src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <AppProvider>
+    <Page title="Judge.Me Reviews">
+      <Layout>
+        <LayoutSection>
+          <Button @click="toggleCollapsibleActive" style="margin-bottom: 20px;">{{ formButtonMessage }}</Button>
+          <Collapsible :open="collapsibleActive" :class="{ collapsibleActive: collapsibleActive }">
+            <Card>
+              <ReviewForm @add-review="addReview" :key="formReset" />
+            </Card>
+          </Collapsible>
+          <Checkbox label="Unverified reviews only" v-model="unverifiedOnly" />
+          <Divider />
+          <EmptyState
+            v-if="!reviews || reviews.length < 1"
```

Comment:
Nit: I would initialise reviews to an empty array, to avoid dealing with `null` values. Then we can simply check `reviews.length`